web/.eslintrc.js

```diff
@@ -1,17 +1,17 @@
 module.exports = {
   root: true,
-  extends: [
-    'eslint:recommended',
-    'plugin:@typescript-eslint/recommended',
-    '@react-native-community',
-    'prettier',
-  ],
+  ignorePatterns: ['.eslintrc.js', 'tsconfig.json', 'package.json'],
```

Comment:
I don't like this, but otherwise I'm getting:

Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: package.json.
The extension for the file (.json) is non-standard. You should add "parserOptions.extraFileExtensions" to your config.

Adding the extraFileExtensions bit only removes that bit of the error.

Any ideas?

Reply:
Nope. :(

this seems fine.